core/src/main/java/com/klaytn/caver/contract/ContractMethod.java

```diff
@@ -0,0 +1,410 @@
+     */
+    List<ContractIOType> outputs;
+
+    /**
+     * The function signature string. ex) foo(uint256,uin256)
+     */
+    String signature;
+
+    /**
+     * The contract address.
+     */
+    String contractAddress;
+
+    /**
+     * The default send option. When you execute call() or send() without SendOptions, defaultSendOptions will be used.
+     */
+    SendOptions defaultSendOptions;
+
+    /**
+     * Creates a ContractMethod instance.
+     */
+    public ContractMethod() {
+    }
+
+    /**
+     * Creates a ContractMethod instance.
+     * @param caver A Caver instance.
+     * @param type The input type. It always set "function"
+     * @param name The function name
+     * @param inputs The list of ContractIOType contains to function parameter information.
+     * @param outputs The list of ContractIOType
+     * @param signature The function signature string.
+     * @param contractAddress The contract address
+     */
+    public ContractMethod(Caver caver, String type, String name, List<ContractIOType> inputs, List<ContractIOType> outputs, String signature, String contractAddress) {
+        this.caver = caver;
+        this.type = type;
+        this.name = name;
+        this.inputs = inputs;
+        this.outputs = outputs;
+        this.signature = signature;
+        this.contractAddress = contractAddress;
+    }
+
+    /**
+     * Execute smart contract method in the EVM without sending any transaction.
+     * It is used defaultSendOption field to sendOptions
+     * @param arguments A List of parameter that solidity wrapper type to call smart contract method.
+     * @return List
+     * @throws NoSuchMethodException
+     * @throws IOException
+     * @throws InstantiationException
+     * @throws IllegalAccessException
+     * @throws InvocationTargetException
+     * @throws ClassNotFoundException
+     */
+    public List<Type> call(List<Object> arguments) throws NoSuchMethodException, IOException, InstantiationException, IllegalAccessException, InvocationTargetException, ClassNotFoundException {
+        return call(arguments, null);
+    }
+
+    /**
+     * Execute smart contract method in the EVM without sending any transaction.
+     * @param arguments A List of parameter that solidity wrapper type to call smart contract method.
+     * @param options An option to execute smart contract method.
+     * @return List
+     * @throws IOException
+     * @throws ClassNotFoundException
+     */
+    public List<Type> call(List<Object> arguments, SendOptions options) throws IOException, ClassNotFoundException, NoSuchMethodException, InstantiationException, IllegalAccessException, InvocationTargetException {
+        List<Object> functionParams = new ArrayList<>();
+
+        if(arguments != null) {
+            functionParams.addAll(arguments);
+        }
+
+        if(options == null) {
+            if(getDefaultSendOptions() == null) {
```

Comment:
If defaultSendOptions can be initiated in Cotract's constructor, this statement doesn't need to check!